src/lib/components/StateSelection.svelte

```
@@ -1,44 +1,18 @@
 <script lang="ts">
-	import * as Select from '$lib/components/ui/select/index.js';
 	import Badge from '$lib/components/ui/badge/badge.svelte';
```

ESLint

[error] 'Badge' is defined but never used.

(@typescript-eslint/no-unused-vars)